frontend/src/components/settings/pages/general/BranchSelect.tsx

```diff
@@ -14,17 +15,23 @@ enum UpdateBranch {
 }
 
 const BranchSelect: FunctionComponent<{}> = () => {
+  const { t } = useTranslation();
+  const tStores = [
+    t('StoreSelect.store_channel.default'),
+    t('StoreSelect.store_channel.testing'),
+    t('StoreSelect.store_channel.custom'),
+  ];
```

Comment:
This is wrong, it is for decky branches not store channels.

Reply:
Whoops, fixing it now.